modules/multiple_evaluator.py

```diff
@@ -0,0 +1,38 @@
+"""
+Module to evaluate multiple Evaluator objects to determine if drone should RTL
+"""
+
+from . import evaluate_condition_abstract
+
+
+class MultipleEvaluator():
+    """
+    Attributes: 
+        evaluate_object_list : A list of EvaluateCondition objects, representing conditions 
+        to evaluate when determining RTL
+    """
+    def __init__(self,
+                 evaluate_object_list : "list[evaluate_condition_abstract.EvaluateCondition]"):
+        """
+        Constructor
+
+        Parameters: 
+            evaluate_object_list : A list of EvaluateCondition objects, representing conditions 
+            to evaluate when determining RTL
+        """
+        self.evaluate_object_list = evaluate_object_list
```

Comment:
Nit: Rename to `condition_list`. When we are naming variables, we don't want to have the data type the variable has since it could change.